src/components/SettingsPane/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import {
+	Box,
+	Card,
+	CardContent,
+	CardHeader,
+	Stack,
+	Typography,
+} from '@mui/material';
+import { color } from '@balena/design-tokens';
+
+export interface SettingsPaneProps {
+	title?: string | React.ReactNode;
+	children: React.ReactNode;
+	danger?: boolean;
+}
+
+export interface SettingsPaneItemProps {
+	title: string | React.ReactNode;
+	description?: string | React.ReactNode;
+	action?: React.ReactNode;
+}
+
+export const SettingsPane = ({
+	title,
+	children,
+	danger,
+}: SettingsPaneProps) => (
+	<Card>
+		{title && (
+			<CardHeader
+				title={title}
+				sx={{
+					'.MuiCardHeader-title': {
+						color: danger ? color.text.danger.value : 'inherit',
+					},
+				}}
+			/>
+		)}
+		<CardContent sx={{ '&:last-child': { pb: 0 } }}>{children}</CardContent>
```

Comment:
Is this `sx` necessary? You have `'&:not(&:last-child)'` on the `SettingsPaneItem` below

Reply:
so I'm not sure why but we have a padding-bottom that's added to `CardContent:last-child`. This removes it (and I have to target `&:last-child` otherwise it gets overriden by the cascade).

Reply:
I looked at the theme but can't find this rule, maybe you have an idea @myarmolinsky?

Reply:
Anyway for the sake of merging this PR I'll add a TODO to find out why we need this